src/Http/ServerRequest.php

```diff
@@ -733,17 +734,18 @@ public function isAll(array $types): bool
      * `addDetector('extension', ['param' => '_ext', 'options' => ['pdf', 'csv']]`
      *
      * @param string $name The name of the detector.
-     * @param callable|array $callable A callable or options array for the detector definition.
+     * @param \Closure|array $callable A Closure or options array for the detector definition.
      * @return void
      */
-    public static function addDetector(string $name, callable|array $callable): void
+    public static function addDetector(string $name, Closure|array $callable): void
```

Comment:
the `$callable` is not proper name right now IMO

Reply:
Yes, I am going to change it to `$detector`. But that will go into `4.next` first as that's where we have done all the argument renames.